test/test_turbulence_closures.jl

```diff
@@ -193,8 +188,14 @@ end
 
     @testset "Closure instantiation" begin
         @info "  Testing closure instantiation..."
-        for closure in closures
-            @test closure_instantiation(closure)
+        for closurename in closures
+            closure = getproperty(TurbulenceClosures, closurename)()
+            @test closure isa TurbulenceClosures.AbstractTurbulenceClosure
+
+            grid = RectilinearGrid(CPU(), size=(1, 1, 1), extent=(1, 2, 3))
+            model = NonhydrostaticModel(grid=grid, closure=closure, tracers=:c)
+            @test diffusivity(model.closure, model.diffusivity_fields, Val(:c)) isa Union{Number, Field, AbstractOperations.AbstractOperation}
+            @test viscosity(model.closure, model.diffusivity_fields) isa Union{Number, Field}
```

Comment:
This could require changes if any closure wants to output a different type... any way we can test functionality rather than type?

Reply:
How about testing that the object it returns can be used within an `AbstractOperation`? like

```julia
op = diffusivity() * ∂x(c); op[1, 1, 1] isa Number
```

Reply:
or even just `op[1, 1, 1] == 0.0` (its a weak test, but better than nothing)

Reply:
Good points. I guess your suggestion already addresses these concerns, right? I just accepted it. Let's see if tests pass